actor/echo/README.md

```diff
@@ -1,15 +1,23 @@
 # {{project-name}}
 
-This actor echoes messages received in an HttpRequest.
-It is linked to the http server provider wasmcloud:httpserver.
+This actor echoes messages received in an `HttpRequest`.
+It is linked to the http server provider `wasmcloud:httpserver`.
 
 For each http request, the actor returns a json-formatted
 string containing fields from the request.
 
+## How do I customize this template to use other contracts & interfaces?
+
+- You can change what contracts this actor claims in `wasmcloud.toml` and the `Makefile`. In the future this will just be in `wasmcloud.toml`.
+- You will then need to change the dependencies in `Cargo.toml` to import the interfaces for the contracts you want. Optionally delete the `wasmcloud-interface-httpserver` dep if you're not using that contract.
+- Finally change the `src/lib.rs` file, changing/deleting the current interface import and `impl` block, while adding a new import & `impl` for any contracts you added!
+
 ### Using the included Github Actions
-If you store your source code on Github, we've gone ahead and included two actions: `build.yml` and `release.yml` under `.github/workflows`. The build action will automatically build, lint, and check formatting for your actor. The release action will automatically release a new version of your actor whenever code is pushed to `main`, or when you push a tag with the form `vX.Y.Z`. 
+
+If you store your source code on Github, we've gone ahead and included two actions: `build.yml` and `release.yml` under `.github/workflows`. The build action will automatically build, lint, and check formatting for your actor. The release action will automatically release a new version of your actor whenever code is pushed to `main`, or when you push a tag with the form `vX.Y.Z`.
```

Comment:
grammar nit: I think this reads more smoothly if we replace the line "If you store ...actions:" with

The generated project include two Github actions:

Reply:
Done!